packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -41,11 +47,19 @@ function toMatchReference(screenshot) {
   if (global.Terra.serviceOptions.useRemoteReferenceScreenshots && !pass
     && (global.Terra.serviceOptions.buildBranch.match(BUILD_BRANCH.pullRequest) || global.Terra.serviceOptions.buildType === BUILD_TYPE.branchEventCause)) {
     pass = true;
-    process.env.SCREENSHOT_MISMATCH_CHECK = true;
-    if (message.length > 0) {
-      message = message.concat('\n');
+    const outputDir = getOutputDir();
+    const fileName = path.join(outputDir, 'ignored-mismatch.json');
+    // Create the output directory if it does not already exist.
+    if (!fs.existsSync(outputDir)) {
```

Comment:
I don't know if it is possible, at the very least I have a suspicion that would be a major version bump since we would be changing a main testing function to be `async` when it wasn't before.